sample-apps/spark/build.gradle.kts

```diff
@@ -33,6 +33,16 @@ jib {
   )
 }
 
+jib {
```

Comment:
how does this works? can you define multiple jib entries?

Reply:
it seems that you cannot https://github.com/GoogleContainerTools/jib/issues/1615#issuecomment-481737593

Reply:
I think you can try to create an empty gradle subproject with just a runtime dependnecy on this project and that configures a jib image. the `build.gradle.kts` would be exactly the same, except that it would have the single runtime dependency on this subproject and the jib section would be for the image without the agent.

Reply:
@rapphil thanks for catching this. I forgot that this issue was pending and incomplete.
I have now updated the gradle build file to include the jib configuration for multiple images.
I have tried to work with a different solution than proposed, as I think it's easier to implement this solution to other sample apps - incase we need to add more images there with different base images.
In this solution, used `.setJibExtension` to construct custom `BuildImageTask` and `JibExtension`.
I tested this solution to push images on multiple repo of my account on local box and it worked.

Reply:
got it. I understand what you did but I think your solution as it is right now will not be able to publish the image `public.ecr.aws/aws-otel-test/aws-otel-java-spark`

you need to delete this block.

Reply:
@rapphil thanks updated accordingly. left that block by mistake after reverting changes of manual testing. deleted it now.